bin/shutdown_dsm.py

```diff
@@ -7,15 +7,16 @@
 
 
 async def shutdown(opts):
-    commands = ['disable', 'standby', 'exitControl']
+    commands = ["disable", "standby", "exitControl"]
```

Comment:
Another good use of `set_summary_state`. It will work from any initial state, and OFFLINE works as expected.

Reply:
That said...I'm curious why you have a shutdown script? I have never seen that done before.

Perhaps we should have a generic one in salobj?

Reply:
I have a shutdown script because the CSC is run locally on a laptop that is connected to hardware. I don't want to have to access LOVE or a Nublado instance to turn it off.

Reply:
Changed to set_summary_state.